packages/component-library/components/Paragraph/Paragraph.tsx

```diff
@@ -19,9 +19,19 @@ export type AllowedTags =
   | "h6"
 
 export interface ParagraphProps {
+  /**
+   * Not recommended. A short-circuit for overriding styles in a pinch
+   * @default ""
+   */
```

Comment:
These are what's displayed in the prop table on the docs tab. The table will still display if these props aren't commented, but it will be missing the description in the props table.